test/cadet_web/admin_controllers/admin_user_controller_test.exs

```diff
@@ -565,6 +565,743 @@ defmodule CadetWeb.AdminUserControllerTest do
     end
   end
 
+  describe "GET /v2/courses/{course_id}/user/total_xp" do
```

Comment:
these are copied over from `user_controller_test.exs`? could we have some tests for the correct admin endpoint?